src/app.js

```diff
@@ -19,15 +21,27 @@ const { getRemoteTabs, readError } = require(path.resolve(
 const pkg = require(path.resolve(__dirname, '../package.json'));
 const notifier = updateNotifier({ pkg });
 
+// Create a Configstore object. If conf file is empty, fill with default values instead and save to config path
```

Comment:
We can probably leave the comments out of the source here. We can add documentation to the readme explaining the config location and configurable options.